raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
`assert not` passes when return is `None`, 0, `''` (which I think may be why it is passing), while `assert api0.manager_address_if_token_registered(token_addresses[0]) is None` only works when return is explicitly `None` (which is what is expected from the docstring).
Anyway, it may be raising an exception in the line 75, when trying to access `.address` from the empty string, and then returning None. But this fail on my next comment, for example.